nautobot/core/middleware.py

```diff
@@ -16,10 +17,24 @@
     remote_auth_enabled,
     ldap_auth_enabled,
 )
+from nautobot.core.utils.config import get_settings_or_config
 from nautobot.core.views import server_error
 from nautobot.extras.context_managers import change_logging, WebChangeContext
 
 
+class LoginRedirectMiddleware(MiddlewareMixin):
+    def __call__(self, request):
+        login_required_paths = ["/graphql/", "/api/docs/"]
```

Comment:
Any suggestions on how I can tell if '/graphql/' is being called from the browser or the API, because the API ought not to redirect?